### PR TITLE
docs: define missing const variables in babel config sample

### PR DESCRIPTION
**Pull request by G-Rath**

### CODE REVIEW

File: docs/customizing_babel_config.md [hunk 1]
```diff
@@ -29,7 +29,9 @@ yarn add --dev @pmmmwh/react-refresh-webpack-plugin react-refresh
 module.exports = function (api) {
   const defaultConfigFunc = require('shakapacker/package/babel/preset.js')
   const resultConfig = defaultConfigFunc(api)
+  const isDevelopmentEnv = api.env('development')
   const isProductionEnv = api.env('production')
+  const isTestEnv = api.env('test')
```
Comment (justin808) [inline docs/customizing_babel_config.md:34]: Thanks for catching this. Maybe we should just have `!isProductionEnv` on line 41?

Comment (justin808) [reply docs/customizing_babel_config.md:34]: CC: @tomdracz

Comment (G-Rath) [reply docs/customizing_babel_config.md:34]: That's not the same though - e.g. if I had a staging or UAT environment, we'd want this disabled as well

**Review by tomdracz — APPROVED** [review]

File: docs/customizing_babel_config.md (hunk 1 above)
Comment (tomdracz) [reply docs/customizing_babel_config.md:34]: LGTM and it shows how to pull different environments so useful to have it like that

**Pull request merged**